webpack.config.ts

```diff
@@ -0,0 +1,112 @@
+import path from "path";
+import sveltePreprocess from "svelte-preprocess";
+import webpack from "webpack";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+
+interface Argv {
+  mode?: "production" | "development";
+}
+
+const tsRule = {
+  test: /\.ts$/,
+  exclude: /node_modules/,
+  use: {
+    loader: "ts-loader",
+    options: {
+      compilerOptions: {
+        noEmit: false,
+        module: "es6",
+      },
+    },
+  },
+};
+
+function electronMain(_env: unknown, argv: Argv): webpack.Configuration {
+  const mode = argv.mode || "development";
```

Comment:
`suggestion`

Have a function to dry the mode fallback and call it directly when setting the `mode` field:

```
  mode: getMode(argv),
```